tox.ini

```diff
@@ -1,44 +1,66 @@
 [tox]
 minversion = 3.7.0
 envlist =
+    build-docker
```

Comment:
This is a list of envs executed when a user runs `tox` without env spec. I'd say that it should be a minimal needed thing for them to verify that a few main things work rather than wait for hours for all of the unneeded things to complete.
So they definitely don't need to check whether docker build works because in 99% cases they won't touch it.

Reply:
If the focus is on the user here, would it make sense to limit this a little more, e.g.
```
envlist =
    doc 
    format-check
    lint
    metadata-validation
    py{27,35,36,37}-ansible{25,26,27}-unit
```

?

Reply:
Let's re-iterate:
1) Do users change docs often? Will they benefit on rebuilding docs all the time? Probably not.
2) format-check? It's probably worth it.
3) metadata-validation? Do they change packaging all the time? Probably not.
4) unit-tests? definitely yes.
5) functional tests? I think so.

Reply:
Agreed on all counts. I left functional testing out because they don't "just work" for me without adding some pytest filtering. Functional testing should probably be in the default list of tests run, but not if those test runs require, say, working credentials for all drivers. I don't know if this is a common problem or only local to me, though.